packages/@ama-sdk/create/src/index.ts

```diff
@@ -65,6 +65,23 @@ if (argv['spec-path'] && argv['spec-package-name']) {
   process.exit(-4);
 }
 
+// TODO: Change to `openapi` in v11 (ref: #1745)
+/* Name of the specification file copied locally (without extension) */
+const LOCAL_SPEC_FILENAME = 'swagger-spec';
```

Comment:
why is it duplicating the constants here, they are already exposed by schematics and core packages which are in dep on create?

Reply:
indeed. ama-sdk/core is a dep here so I could import the constants from there